ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -80,16 +80,39 @@
     - restart daemons
 
 - include: gen_tokens.yml
-  when: inventory_hostname == groups['masters'][0]
```

Comment:
There is/was also a problem that when using "when" for the include that the other masters only got the `gen_tokens` var with "empty" (skipped) values.